Library/Formula/openssl.rb

```diff
@@ -121,6 +118,33 @@ def caveats; <<-EOS.undent
     EOS
   end
 
+  private
+
+  def get_system_certificates
+    keychains = %w[
+      /Library/Keychains/System.keychain
+      /System/Library/Keychains/SystemRootCertificates.keychain
+    ]
+
+    certs_list = `security find-certificate -a -p #{keychains.join(" ")}`
+    certs = certs_list.scan(
+      /#{Regexp.escape('-----BEGIN CERTIFICATE-----')}.*?#{Regexp.escape('-----END CERTIFICATE-----')}/m
```

Comment:
There aren't any metacharacters here so we shouldn't need to escape the strings.